oak-core/src/main/java/org/apache/jackrabbit/oak/Oak.java

```diff
@@ -974,6 +975,10 @@ public void setStrictPathRestriction(String strictPathRestriction) {
             settings.setStrictPathRestriction(strictPathRestriction);
         }
 
+        public void setPrefetchFeature(@Nullable Feature prefetch) {
```

Comment:
This method is just a wrapper of this one, so that's why I think it is more appropriate to maintain the same definition, as it is public and could be used from other place at some point:

https://github.com/apache/jackrabbit-oak/blob/trunk/oak-core/src/main/java/org/apache/jackrabbit/oak/query/QueryEngineSettings.java#L134